__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const { topicData, userData, articleData, commentData } = require('../db/data/test-data/');
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
```

Comment:
You can just require and pass this in as data as its already being destructed in the seed file 🙂